Remotion/ObjectBinding/Web/UI/Controls/BocTreeNode.cs

```diff
@@ -91,14 +94,16 @@ public IBusinessObjectWithIdentity BusinessObject
   ///   The <see cref="IBusinessObjectReferenceProperty"/> that was used to access the 
   ///   <see cref="IBusinessObjectWithIdentity"/> of this <see cref="BusinessObjectTreeNode"/>.
   /// </summary>
+  [AllowNull]
```

Comment:
Null can be set, but not retrieved. Am I wrong?

Reply:
I'm not sure. EnsureProperty is an IF/ELSEIF and I think you could construct something that has a TreeNode that's not covered by this. 

It may well be that there's going to be a null-ref exception somewhere if you do something like this, but I can't promise the property contract without a deepdive. It's probably a nullbug, but we should go with nullable for now, for Property and add a bugfix-todo for this.

Reply:
This case is currently guarded by an assertion in the last line of EnsureProperty

Reply:
Ah, yes. I think this guard's a bug until proofen otherwise. I'll add a remark for this.